yarn-project/end-to-end/src/fixtures/utils.ts

```diff
@@ -279,12 +285,7 @@ export async function setup(
     opts.deployL1ContractsValues ?? (await setupL1Contracts(config.rpcUrl, hdAccount, logger));
 
   config.publisherPrivateKey = `0x${publisherPrivKey!.toString('hex')}`;
-  config.l1Contracts.rollupAddress = deployL1ContractsValues.l1ContractAddresses.rollupAddress;
-  config.l1Contracts.registryAddress = deployL1ContractsValues.l1ContractAddresses.registryAddress;
-  config.l1Contracts.contractDeploymentEmitterAddress =
-    deployL1ContractsValues.l1ContractAddresses.contractDeploymentEmitterAddress;
-  config.l1Contracts.inboxAddress = deployL1ContractsValues.l1ContractAddresses.inboxAddress;
-  config.l1Contracts.outboxAddress = deployL1ContractsValues.l1ContractAddresses.outboxAddress;
+  config.l1Contracts = deployL1ContractsValues.l1ContractAddresses;
 
```

Comment:
does this need to be updated in the sandbox too? We do something similar there to show L1 addresses for L1<>L2 messaging

Reply:
What do you mean? The update here is mainly cosmetic? Just taking all at once instead of copying one by one. 

```ts
a.a = b.a;
a.b = b.b;
a.c = b.c;
```

is replaced with
```ts
a = b;
```